source/event.c

```diff
@@ -1158,10 +1159,9 @@ static struct gp_event *get_gpe(
     uacpi_namespace_node *gpe_device, uacpi_u16 idx
 )
 {
-    struct gpe_search_ctx ctx = {
-        .gpe_device = gpe_device,
-        .idx = idx,
-    };
+    struct gpe_search_ctx ctx = {0};
```

Comment:
I usually do `{ 0 };` (note the spaces), and one empty line between declarations and other code